### PR TITLE
V3: 183016994 dataset switching popup

### PR DESCRIPTION
**Pull request by eireland**
Adds ability to switch data sets via dropdown

### CODE REVIEW

File: v3/src/components/data-summary.tsx [hunk 1]
```diff
@@ -28,9 +30,34 @@ export const DataSummary = observer(({ broker, v2Document }: IProps) => {
...
+        <Select onChange={handleDataSetSelection} value={data?.id}>
+          { dataSetSummaries?.map(summary => {
+              return renderOption(summary.name || `DataSet ${summary.id}`, summary.id)
+            })
+          }
+        </Select>
+      )
+    } else {
+      return null
+    }
```
Comment (kswenson) [inline v3/src/components/data-summary.tsx:54]:
```suggestion
    return null
```
You don't need the `else` here.

**Review by kswenson — CHANGES_REQUESTED** [review]
👍 Looks good -- just a couple suggestions

**Review by bfinzer — APPROVED** [review]
👍🏻 LGTM

**Review by kswenson — APPROVED** [review]
👍 Changes look good

**Pull request merged**